code/modules/cm_marines/vehicle_part_fabricator.dm

```diff
@@ -12,6 +12,20 @@
 	var/generate_points = TRUE
 	var/omnisentry_price_scale = 100
 	var/omnisentry_price = 300
+	var/list/datum/build_queue_entry/build_queue = list()
+
+/datum/build_queue_entry
+	var/item
+	var/cost
+
+/datum/build_queue_entry/New(item, cost)
+	src.item = item
+	src.cost = cost
+
+/obj/structure/machinery/part_fabricator/examine(mob/user)
```

Comment:
use get_examine_text, examine() isn't for this anymore (because of the examine blocks)

Reply:
Changed